test/framework/kogitomanagementconsole.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package framework
+
+import (
+	"fmt"
+
+	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/client/kubernetes"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/framework"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/infrastructure"
+	apps "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// InstallKogitoManagementConsole deploy the Kogito Management Console
+func InstallKogitoManagementConsole(namespace string, installerType InstallerType, replicas int) error {
+	GetLogger(namespace).Infof("%s install Kogito Management Console with %d replicas", installerType, replicas)
+	switch installerType {
+	case CRInstallerType:
+		return crInstallKogitoManagementConsole(namespace, replicas)
+	default:
+		panic(fmt.Errorf("Unknown installer type %s", installerType))
+	}
+}
+
+func crInstallKogitoManagementConsole(namespace string, replicas int) error {
+	service := getManagementConsoleStub(namespace, int32(replicas))
+
+	if _, err := kubernetes.ResourceC(kubeClient).CreateIfNotExists(service); err != nil {
+		return fmt.Errorf("Error creating Kogito Management Console: %v", err)
+	}
+	return nil
+}
+
+func getManagementConsoleImageTag() v1alpha1.Image {
+	if len(GetConfigManagementConsoleImageTag()) > 0 {
```

Comment:
should call `config.GetManagementConsoleImageTag()` now :)

Reply:
Fixed.